proto/cosmos/crisis/v1/tx.proto

```diff
@@ -0,0 +1,97 @@
+syntax = "proto3";
+package cosmos.crisis.v1;
+
+import "cosmos/msg/v1/msg.proto";
+
+// Msg defines the crisis Msg service.
+service Msg {
+  option (cosmos.msg.v1.service) = true;
+
+  // AuthorizeCircuitBreaker allows a super-admin to grant (or revoke) another
+  // account's circuit breaker permissions.
+  rpc AuthorizeCircuitBreaker(MsgAuthorizeCircuitBreaker) returns (MsgAuthorizeCircuitBreakerResponse);
```

Comment:
Who would be the authorizer to start? 

I think we should make a version that works with 0.45/46 and beyond.

Reply:
The authorizer but by the gov authority param. We can also add legacy gov content proposals for this.

Reply:
wouldn't be simpler just to have a single address that has the circuit breaker permissions? The address could then be set by each network / application: some may set it to x/gov's module address, others may have a dedicated x/group, etc.?

Reply:
for example, that `LEVEL_SOME_MSGS` vs `LEVEL_ALL_MSGS` can be configured via a single group's policies?

Reply:
There were some discussions about specific module maintainers getting permission their modules across multiple networks. @tac0turtle maybe you can speak to use cases. We should definitely loop in other folks with context here.

Reply:
> wouldn't be simpler just to have a single address that has the circuit breaker permissions? The address could then be set by each network / application: some may set it to x/gov's module address, others may have a dedicated x/group, etc.?

yea this could be done via authz, or gov proposal to migrate complete control to a group. 


module maintainer approach is scary for me. That would put a huge burden on the core sdk team.

Reply:
For circuit breaker we probably want to provide built in ways to do simple authorization because what if the bug is in authz or group?

Reply:
> For circuit breaker we probably want to provide built in ways to do simple authorization because what if the bug is in authz or group?

Yes, agreed. Because you could end up circuit-breaking authz or groups itself. The original super-admin should probably be the gov module as stated. After that, you can have each module have it's own authority, right?

Reply:
> After that, you can have each module have it's own authority, right?

If maintainers were willing to do that sure

Reply:
I think I concur with @tomtau to have a simpler design in x/crisis, i.e. one single circuit breaker address that can trip all Msgs. We don't need Authorize anymore in this case, just Trip and Reset.

Then, optionally, this single address can create authz grants to other addresses.

```proto
message CircuitBreakerAuthorization {
  // msg_type_urls are Msgs that the grantee can trip. If left empty, then
  // the grantee can trip **ALL** Msgs.
  repeated string msg_type_urls = 2;
}
```

And the only difference is that grantees would need to call `MsgExec{MsgTrip}` instead of `MsgTrip` directly with the current design.